srcpkgs/faust/template

```diff
@@ -0,0 +1,52 @@
+# Template file for 'faust'
+pkgname=faust
+version=2.20.2
+revision=1
+build_style=gnu-makefile
+hostmakedepends="cmake pkg-config llvm"
+makedepends="libmicrohttpd-devel llvm-libunwind-devel"
+short_desc="Functional Programming Language for Real Time Signal Processing"
+maintainer="mag <mag-one@autistici.org>"
+license="GPL-2.0-or-later"
+homepage="http://www.faust.grame.fr/"
+distfiles=https://github.com/grame-cncm/faust/releases/download/${version}/${pkgname}-${version}.tar.gz
+checksum="c32588ee60d0f7ca25034d3d5436def6b35495f4d3beb3b02208d48684bcfbe9"
+nostrip_files="libsndfile.so "
+noshlibprovides="libsndfile.so"
+
+case $XBPS_TARGET_MACHINE in
+	*-musl) makedepends+=" libexecinfo-devel";;
+esac
+
+# INFO:
+#
+# - Faust puts its own libraries in /usr/share/faust
+# - In the same dir there's the Android app template
+#   used by the 'faust2android' script.
+# - The Android app template contains 'libsndfile.so'
+#   for Android that causes the installation to fail:
+#
+# => faust-2.15.11_1: running post-install hook: 11-pkglint-elf-in-usrshare ...
+# => ERROR: faust-2.15.11_1: ELF files found in /usr/share:
+# => ERROR:    /usr/share/faust/android/app/lib/libsndfile/lib/armeabi-v7a/libsndfile.so
+# => ERROR:    /usr/share/faust/android/app/lib/libsndfile/lib/arm64-v8a/libsndfile.so
+# => ERROR: faust-2.15.11_1: cannot continue with installation!
+#
+
+# FIRST (LAZY) WORKAROUND:
+#
+#post_install() {
+# rm -rf $PKGDESTDIR/usr/share/faust/android/
+#}
+
+# BETTER (?) WORKAROUND:
+#
+skiprdeps="/usr/lib/faust/android/app/lib/libsndfile/lib/armeabi-v7a/libsndfile.so /usr/lib/faust/android/app/lib/libsndfile/lib/arm64-v8a/libsndfile.so"
+
+post_install() {
+	mv $PKGDESTDIR/usr/share/faust $PKGDESTDIR/usr/lib
```

Comment:
As one mentioned in other PR why rm and then link?

Reply:
I move /usr/share/faust in /usr/lib (or the '11-pkglint-elf-in-usrshare' hook will fail)
then i link /usr/share/faust to /usr/lib/faust (because faust will expect his libraries/tools in /usr/share...)
  
'rm'? sorry...not sure I understood...

Reply:
The last [commit](https://github.com/void-linux/void-packages/pull/17568/commits/888ee70f27615bfffe1eb116ed6d451b613357f0) should solve this. The link is no longer necessary.

I had to patch the sources cause there's no way to set that path in build scripts (or I can't see it).